### DISCUSSION TITLE
Still maintained?

### DISCUSSION THREAD

**Discussion by barnou-psee** [JakubAndrysek/MkDoxy#117] [General]
Hello, I wonder if this repo is still maintained? I see last issues / PRs are left as is without any activity from the maintainer (I opened one myself).

**Comment by JakubAndrysek** [comment]
Hi, sorry for my inactivity. I worked on different projects. I will look at PRs today, check them and merge them.
Thank you for the reminder.

**Reply by barnou-psee** [reply]
I didn't want to raise that much urgency, just wanted to know if you were still maintaining the repo. Do you need help on it?

**Reply by JakubAndrysek** [reply]
I have started this project many years ago. During that time, the community that uses this plugin has grown a lot.
If you have time, I'd be happy to help with PRs and documentation renewal, for example.

**Reply by barnou-psee** [reply]
That's a good sign ;) I can try to help, but it might be only during the weekend when I'm not using the plugin as I'm working I guess just like you :sweat_smile:
